_dev/sources/miscBuilder.ts

```diff
@@ -24,22 +27,33 @@ export class MiscBuilder {
         Here: "HERE Maps",
         MapBox: "MapBox"
     };
+    private purgeSettings: any;
+    private emailSenderFrom: string;
+    private customerClassification: string;
 
     private abortCurrentTask () {
         this.currentTask && this.currentTask.abort && this.currentTask.abort();
         this.currentTask = null;
     }
 
+    //todo problem code...is this necessary?
     private getAllowedAddins (allAddins: string[]) {
         return allAddins.filter(addin => {
             let addinConfig = JSON.parse(addin);
             return addinConfig && Array.isArray(addinConfig.items) && addinConfig.items.every(item => {
                 let url = item.url;
+                console.log('here...');
                 return url && url.indexOf("\/\/") > -1;
             });
         });
     }
 
+    private removeExportAddin (allAddins: string[]) {
```

Comment:
Merge this method into getAllowedAddins

Reply:
yes sure...will first resolve the issue above.

Reply:
done